src/middlewares/validators/nameCategoryPayload.js

```diff
@@ -0,0 +1,16 @@
+const Joi = require('joi');
+
+const nameCategoryPayloadValidator = (req, res, next) => {
```

Comment:
What a strange naming? Name it just "categoryPayloadValidator". What if there are more fields, like in user?) Will you name it nameSurnameAge...UserValidator?

Reply:
Haha I agree with you, Vasyl. Fixed.